server/routes/index.js

```diff
@@ -1,9 +1,9 @@
-const express = require('express');
-const router = express.Router();
+var express = require("express");
+var router = express.Router();
```

Comment:
This doesn't seem right.

cc @JChris246 did the linting rules say anything about `const` vs `let` vs `var`?

Reply:
ESLint generally discourages the use of `var` mainly to avoid the accidental overriding of previously defined variables. The better choice in this situation would be to use `const` as I don't believe we would want the value of `express` to change given that it's an import.